### PR TITLE
T/T Database Information Update

### PR DESCRIPTION
**Pull request by sniper7kills**
Updates the `hooks-database`, `database-drivers`, `database-mysql`, and `database-sqlite` documentation to provide an overview and fix inconsistencies between multiple pieces of documentation where the same information was presented in different ways.

Added cross links between documents to ensure information was easily accessible.

### CODE REVIEW

File: docs/tenancy/1.x/affects-connections.md [hunk 1]
```diff
@@ -99,3 +139,11 @@ class Role extends Model
...
 }
 ```
+
+### Laravel DB Facade Example
+
+If you are using Laravel's `DB` Facade, you are able to access the tenant connection as seen in the example below.
+
+```php
+DB::connection('tenant')->select(...);
+```
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/affects-connections.md:149]: 3 things:
- Let's add a small notice stating that this is not the recommended way.
- Let's change the example to use the facade to get the connection name `Tenancy::getTenantConnectionName()`
- Let's show that you can also get this connection from the Tenancy Facade / Environment `::getTenantConnection()`

File: docs/tenancy/1.x/database-drivers.md [hunk 2]
```diff
@@ -8,10 +8,25 @@ tags:
...
+Each Database Driver will have their own installation guide.
+
+### First Party Drivers
 
 - [MySQL](database-mysql)
 - [SQLite](database-sqlite)
+
+### Second Party Drivers
+
+Check back soon!
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/database-drivers.md:32]: Why not third party? And not sure I would want to list these in the docs for now.

File: docs/tenancy/1.x/database-mysql.md [hunk 3]
```diff
@@ -9,16 +9,65 @@ tags:
...
+- and Many More!
+
+## Requirements
+
+Because this package runs specific "elevated permissions" queries in order to provide a database and a database user, the user defined in `database.php` needs to have the ability to create users, databases, and grant privileges.
+
+The following is a sample query to create a user with those privileges. 
+
+```mysql
+TODO
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/database-mysql.md:46]: You can "steal" this from the hyn docs.

File: docs/tenancy/1.x/database-mysql.md [hunk 4]
```diff
@@ -9,16 +9,65 @@ tags:
...
+## Overview
+
+**Purpose**
+
+The purpose of this package is to enable the management (Creation, Updating, Deletion) of Tenant MySQL Databases and MySQL Users.
+
+**Requirements**
+
+When using the [`hooks-database`](hooks-database) package the MySQL user defined in `database.php` must have the ability to create users, databases, and grant privileges.
+
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/database-mysql.md:28]: Affects Connections is a direct dependency here.

File: docs/tenancy/1.x/database-sqlite.md [hunk 5]
```diff
@@ -22,3 +55,18 @@ This package will allow you to use the `hooks-database` for SQLite Database inst
...
 
 > CAUTION: This database driver might require additional steps for configuration. It is based on a filesystem, and not a real database like most Database Drivers.
+
+## Configuration
+
+Detailed configuration steps are located in the [`hooks-database`](hooks-database) package and the [`affects-connections`](affects-connections) package.
+
+### Example
+
+In the following example we set the connection to use SQLite, and set the database path to be specific to the tenant based on the Tenant's ID.
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/database-sqlite.md:65]: Let's use in the tenantKey instead, it's a contract function so feels clearer / better

File: docs/tenancy/1.x/database-mysql.md (hunk 3 above)
Comment (sniper7kills) [reply docs/tenancy/1.x/database-mysql.md:46]: I think I'm going to look a little more into this one before I just "steal" it from hyn

File: docs/tenancy/1.x/database-mysql.md [hunk 6]
```diff
@@ -24,12 +24,14 @@ The purpose of this package is to enable the management (Creation, Updating, Del
...
-When using the [`hooks-database`](hooks-database) package the MySQL user defined in `database.php` must have the ability to create users, databases, and grant privileges.
+- MySQL server with a version >= `5.7.6`
+- When using the [`hooks-database`](hooks-database) package the MySQL user defined in `database.php` must have the ability to create users, databases, and grant privileges.
+- This package also required the installation and configuration of the [`affects-connections`](https://tenancy.dev/docs/tenancy/1.x/affects-connections) package. 
+  (*Note:* This package is not required for the use of the [`affects-connections`](affects-connections) package)
 
 **Recommendations**
 
 - To automate the management of Tenant Databases when a Tenant is created, updated, or deleted; The [`hooks-database`](hooks-database) package should also be installed.
-- To use the Tenant Database with the `onTenant` trait; The [`affects-connections`](https://tenancy.dev/docs/tenancy/1.x/affects-connections) package should also be installed. (*Note:* This package is not required for the use of the [`affects-connections`](affects-connections) package)
```
Comment (ArlonAntonius) [inline docs/tenancy/1.x/database-mysql.md]: Why was this removed?

Comment (sniper7kills) [reply docs/tenancy/1.x/database-mysql.md]: Because its *required* now; so the recommendation is a mute point.
